std/http/client.ts

```diff
@@ -0,0 +1,19 @@
+/** Http request built on client side */
+export interface ClientRequest {
```

Comment:
Is this only used for testing? Why not use fetch?

Reply:
`fetch` support keep-alive connection?  I needed `Conn` based req/res for checking connection closed after keep-alive timeout passed. And there are a lot of raw-level http protocol writing in http tests. I want to cleanup them gradually.